reports/helmcharts/helm-chart-versions.sh

```diff
@@ -130,12 +129,14 @@ for chart in $(echo "$charts" | jq -c '.[]'); do
   fi
 
   # Enhance document with additional information
-  uuid=$(uuidgen)
   created_on=$(date '+%Y-%m-%d %H:%M:%S')
+  uuid=$(uuidgen)
+  id="$(uuidgen)"
```

Comment:
Dont need both lines, they do the same thing so just pick whichever you want and use it for the rest of the script.